src/js/components/picklist/picklist.scss

```diff
@@ -62,6 +62,15 @@
   }
 }
 
+.picklist__item-content {
+  display: flex;
+  flex-grow: 1;
+
+  .icon {
+    margin-top: 2px;
```

Comment:
I don't love doing `2px` here. But not sure how else to align these properly. Can't do `align-items: center` because the icon needs to stay aligned with the first line of text when there are multiple overflowing lines.